lighthouse-core/computed/metrics/lantern-largest-contentful-paint.js

```diff
@@ -26,7 +25,18 @@ class LanternLargestContentfulPaint extends LanternMetric {
   }
 
   /**
-   * TODO: Validate.
+   * @param {Node} node
+   * @return {boolean}
+   */
+  static isLowPriorityImageNode(node) {
```

Comment:
probably good to have a comment somewhere on why this distinction (and maybe link back to the original issue). This method might be a good place for it?

Reply:
done